src/System.Private.ServiceModel/tests/Common/Infrastructure/xunit/WcfTestCase.cs

```diff
@@ -2,7 +2,10 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System;
+using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Diagnostics.Tracing;
```

Comment:
Does this compile on N?  The fact you added new dependencies may mean we need to update the ToF wcfopen.settings.targets or make this code conditional in N.  If we make any conditional code in these files, we will need to think about how the appropriate $(DefineConstant) gets set.   I don't think FEATURE_NETNATIVE is set when building tests except in ToF.